src/constants.js

```diff
@@ -20,6 +20,10 @@ module.exports = {
   // subject find, use with config.refocus.url
   findSubjectsEndpoint: '/v1/subjects',
 
+  // fraction of heartbeat interval time before abandoning the request.
+  // a value of 2 would be 1/2 of heartbeat interval time, 3 is 1/3, etc.
```

Comment:
i think it would be clearer as a percentage, e.g. 0.5 for 50% of the heartbeat interval time, 0.25 for a quarter of the heartbeat interval time, etc.